src/connectors/connector_cloudfunctions.workflows.json

```diff
@@ -15,10 +15,10 @@
               "name": "example-function"
             },
             {
-              "sourceArchiveUrl": "gs://your-bucket/cloud-function-source.zip"
+              "sourceArchiveUrl": "gs://BUCKET_NAME/cloud-function-source.zip"
```

Comment:
do we think BUCKET_NAME seem like it might be confused with ${BUCKET_NAME}? a user thinking this is a var vs a field they need to complete?

Reply:
BUCKET_NAME is a placeholder for the variable (and that's indicated in the YAML comment)